packages/gatsby/lib/utils/webpack.config.js

```diff
@@ -269,13 +263,21 @@ module.exports = async (
   }
 
   function resolve() {
+    const { program } = store.getState()
     return {
       // use the program's extension list (generated via the 'resolvableExtensions' API hook)
       extensions: [``, ...program.extensions],
       // Hierarchy of directories for Webpack to look for module.
-      // First is the site directory.
+      // First is the site root directory.
+      // Then all directories needed for building static site.
       // Then in the special directory of isomorphic modules Gatsby ships with.
-      root: [directory, path.resolve(__dirname, `..`, `isomorphic`)],
+      root: [
```

Comment:
Please revert this. I don't want multiple roots as in practice this is quite confusing. Just src should be a root now.